broker-core/src/main/java/org/apache/qpid/server/model/adapter/FileBasedGroupProviderImpl.java

```diff
@@ -359,16 +359,21 @@ protected void onOpen()
             if (childClass == GroupMember.class)
             {
                 String memberName = (String) attributes.get(GroupMember.NAME);
-
-                _groupDatabase.addUserToGroup(memberName, getName());
-                UUID id = UUID.randomUUID();
-                Map<String,Object> attrMap = new HashMap<String, Object>();
-                attrMap.put(GroupMember.ID,id);
-                attrMap.put(GroupMember.NAME, memberName);
-                GroupMemberAdapter groupMemberAdapter = new GroupMemberAdapter(attrMap);
-                groupMemberAdapter.create();
-                return Futures.immediateFuture((C) groupMemberAdapter);
-
+//                Set<String> users = _groupDatabase.getUsersInGroup(getName());
```

Comment:
commented code....

the changes in this hunk needs to be reverted

Reply:
Done!

Reply:
Done!